### PR TITLE
[15.0][MIG] stock_no_negative

### PR DESCRIPTION
**Pull request by JoanMForgeFlow**
Standard migration to version 15.0.

### CODE REVIEW

File: stock_no_negative/tests/test_stock_no_negative.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestStockNoNegative(TransactionCase):
+    at_install = False
+    post_install = True
+
+    def setUp(self):
```
Comment (rousseldenis) [inline stock_no_negative/tests/test_stock_no_negative.py]: I would have done in a class method.

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review.

File: stock_no_negative/tests/test_stock_no_negative.py (hunk 1 above)
Comment (JoanMForgeFlow) [reply stock_no_negative/tests/test_stock_no_negative.py]: If I understood correctly, these changes should be correct now, could you please confirm it? thank you

Comment (pedrobaeza) [reply stock_no_negative/tests/test_stock_no_negative.py]: I think there's no need of the post_install flag for being correctly tested in this case, as it was without it on 14.0.

File: stock_no_negative/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# ?? 2015-2016 Akretion (http://www.akretion.com)
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# ?? 2016 ForgeFlow S.L.
```
Comment (AaronHForgeFlow) [inline stock_no_negative/tests/__init__.py:3]:
```suggestion
# Copyright 2016 ForgeFlow S.L.
```
And with the others

File: stock_no_negative/tests/test_stock_no_negative.py (hunk 1 above)
Comment (JoanMForgeFlow) [reply stock_no_negative/tests/test_stock_no_negative.py]: Ok! :+1:

File: stock_no_negative/tests/__init__.py (hunk 2 above)
Comment (JoanMForgeFlow) [reply stock_no_negative/tests/__init__.py:3]: solved!

**Comment by LoisRForgeFlow** [comment]
@rousseldenis Could you update your review?

File: stock_no_negative/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -1,6 +1,7 @@
 * Alexis de Lattre <alexis.delattre@akretion.com>
 * ForgeFlow S.L. <contact@forgeflow.com>
   * Jordi Ballester
+  * Joan Mateu
 * Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
 * Tecnativa <https://www.tecnativa.com>
   * Pedro M. Baeza
```
Comment (LoisRForgeFlow) [inline stock_no_negative/readme/CONTRIBUTORS.rst]: This is formatted wrongly:

image

Could you fix it?

**Review by rousseldenis — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**